pages/your_tasks/data_workflow_diagram.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Data workflow diagram
+contributors: [Vilem Ded]
+description: Best practices to capture your planned data workflow in a diagram.
+page_id: data_workflow_diagram
+related_pages: 
```

Comment:
```suggestion
related_pages: 
   your_tasks: [dmp]
```

Reply:
Cyrrentlyw e do not allow task pages to be linked to other task pages. Only assmeblies: https://rdmkit.elixir-europe.org/editorial_board_guide#related-pages

Reply:
ok. so we cannot link it to DMP task